docs/api.md

```diff
@@ -125,3 +130,31 @@ Response on success:
   "status": "success"
 }
 ```
+
+## Ready / Health API
+
+### Readiness Check
+
+```
+GET /-/ready
+```
+
+Status code: 200 on success. 
```

Comment:
nitpick: Rephrase this and the health endpoint blurb to something like `Status code: 200 if ready/healthy"`. "on success" is a little weird to me.